app/components/channel_drawer/channel_drawer.js

```diff
@@ -34,9 +48,12 @@ export default class ChannelDrawer extends PureComponent {
     };
 
     state = {
-        openDrawer: false
+        openDrawer: false,
+        openDrawerOffset: 40
     };
 
+    swiperIndex = 1;
```

Comment:
Shouldn't this be part of state? It looks like both ViewPagerAndroid and Swiper take the index as a prop